openedx/core/djangoapps/user_api/accounts/tests/test_views.py

```diff
@@ -1126,49 +1127,95 @@ def test_update_account_settings_rollback(self, mock_email_change):
 class NameChangeViewTests(UserAPITestCase):
     """ NameChangeView tests """
 
-    def setUp(self):
-        super().setUp()
-        self.url = reverse('name_change')
+    def _send_create(self, client, json_data, expected_status=201):
```

Comment:
organizational nit - this helper method sends a request and then asserts a response

but when you read it in other places in code you can't see that it is in fact doing a test, it looks like it just throwing a request out and testing nothing

I'd suggest naming them something like _assert_create_response or _check_create or some other name that tells the reader the code performs a test

alternately just have it return the status code and then assertEqual(_sendCreate(), status) to put the assert right there